### DISCUSSION TITLE
Bug?

### DISCUSSION THREAD

**Discussion by ItzAlmaZ** [BEEmod/BEE2-items#4207] [Tech Support / Q&A]
https://user-images.githubusercontent.com/104623098/183913349-5060ceb1-fb1e-4583-8c37-0747de656446.mp4

In this video you can see the bug that made the automatic portal invisible on the portal 1 style. On Clear it is visible. (And the texture of the Retractable pedestrian button disappeared somewhere) I have no idea how the bug is caused. Maybe reinstalling bee mod will help.

**Comment by ItzAlmaZ** [comment]
Reinstalling didn't help

**Comment by PlatypusBEEPBEEP** [comment]
Any bugs you have about items should be reported on the issues section [here](https://github.com/BEEmod/BEE2-items/issuesl).
